.github/workflows/rw-doc-release-apidocs.yml

```diff
@@ -56,27 +49,33 @@ jobs:
             git config --global user.email "git-action@gooddata.com"
             git config --global user.name "git-action"
 
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: 18.17.0
+      - name: Setup rush
+        env:
+          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        uses: ./gooddata-ui-sdk/.github/actions/rush/set-up-rush
 
-      - name: Install rush
+      - name: Build sdk
         run: |
-            npm install -g @microsoft/rush
+          # we need checkout and build sdk first we have to checkout api docs in to subfolder after rush work done because new git in subfolder will rush fail
+          # we need reconstruct api-extractor temp files, that are used in the build of apidocs and are not cached becaouse they are quite big
+
+          node common/scripts/install-run-rush.js build
+          node common/scripts/install-run-rush.js api-extractor
```

Comment:
suggestion:
run api-extractor to make sure we have the api jsons generated in case the build is restored from the cache